src/node_errors.cc

```diff
@@ -2,14 +2,68 @@
 #include "node_internals.h"
 #include "v8.h"
 
-using v8::HandleScope;
+using v8::Array;
```

Comment:
I'd rather use explicit full names.
It even seems for such a small file it will save text.